app/products/calls/hooks.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// Check if calls is enabled. If it is, then run fn; if it isn't, show an alert and set
+// msgPostfix to ' (Not Available)'.
+import {useState} from 'react';
+import {useIntl} from 'react-intl';
+import {Alert} from 'react-native';
+
+import {useServerUrl} from '@context/server';
+import NetworkManager from '@managers/network_manager';
+
+export const useTryCallsFunction = (fn: () => void) => {
+    const {formatMessage} = useIntl();
+    const serverUrl = useServerUrl();
+    const client = NetworkManager.getClient(serverUrl);
```

Comment:
to be addressed in a later PR if you want, but `getClient` can throw if there is no client registered for the url

Reply:
Ah, good point. Fixed.